tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+skipsdist=True
+envlist =
+    py{36,37,38}-django2.2
+    py{36,37,38}-django3.0
+
+[testenv]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+setenv =
+    PYTHONWARNINGS=default
+    COVERAGE_FILE=.coverage.tox.{envname}
+    TOXENV={envname}
+    DJANGO_REDIS_HOST: localhost
+    py{36,37,38}-django2.2: TOX_DJANGO_VERSION=django>=2.2, <3.0
+    py{36,37,38}-django3.0: TOX_DJANGO_VERSION=django>=3.0, <3.1
```

Comment:
Does it make sense for us to test multiple versions of django here?

Reply:
It was put in before the 2/3 discussion, could probably be stripped back to 3/latest now if that's the plan?